### ISSUE TITLE
Error with xgx_scale_x_log10() 

### ISSUE THREAD

**Issue by iamstein** [Novartis/xgxr#53]
When all x data in a dataset (or even a facet) is zero, xgx_scale_x_log10() throws an error while scale_x_log10() does not. 

```
library(ggplot2)
library(xgxr)
data = data.frame(x = c(0,0), y = c(1,2))

#this line throws an error, using xgx_scale_x_log10()
ggplot(data, aes(x = x, y = y)) + geom_point() + xgx_scale_x_log10()

#this line works, and throws a warning, using scale_x_log10
ggplot(data, aes(x = x, y = y)) + geom_point() + scale_x_log10()
```

*Error* 
Error in seq.default(from = dmin, to = dmax, length.out = m) : 
  'from' must be a finite number

*Cause of error*
I expect when getting the xtick labels, it uses the seq function and if the min and max are the same, it might cause an issue.  Thus we might need an if statement to catch this possibility to prevent the error.

*Package versions*
 - R version 4.1.0
 - ggplot2 version 3.3.5
 - xgxr version 1.1.1

Thanks to Anwesha Chaudhury for finding this issue!

**Comment by margoal1** [comment]
The error appears to be related to xgx_breaks_log10() and xgx_minor_breaks_log10(), as setting breaks and minor_breaks manually removes the error. Setting labels manually does not remove the error.

```
# No more error
ggplot(data.frame(x = c(0,0), y = c(1,2)), aes(x = x, y = y)) + geom_point() + 
       xgx_scale_x_log10(breaks = 1, minor_breaks = 1)

# Still has error
ggplot(data.frame(x = c(0,0), y = c(1,2)), aes(x = x, y = y)) + geom_point() + 
       xgx_scale_x_log10(labels = 1)
```

**Comment by iamstein** [comment]
Resolved issue in version 1.1.4
